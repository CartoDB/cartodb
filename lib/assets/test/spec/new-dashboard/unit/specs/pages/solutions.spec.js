import { shallowMount } from '@vue/test-utils';
import SolutionsPage from 'new-dashboard/pages/Solutions';

const $t = key => key;

describe('Solutions.vue', () => {
  it('should render correct contents', () => {
    const page = shallowMount(SolutionsPage, {
      mocks: {
        $t
      }
    });

    expect(page).toMatchSnapshot();
  });
});
