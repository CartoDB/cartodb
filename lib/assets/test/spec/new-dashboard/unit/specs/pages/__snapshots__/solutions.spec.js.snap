// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Solutions.vue should render correct contents 1`] = `
<section class="page is-bgSoftBlue">
  <div class="container grid">
    <h3 class="title is-body solutions-title">SolutionsPage.solutionsTitle</h3>
    <p class="text is-caption solutions-description">SolutionsPage.solutionsDescription</p>
    <ul class="grid-cell--col12 grid">
      <li class="grid-cell--col12 solution-element">
        <solutioncard-stub title="SolutionsPage.sitePlanning.title" description="SolutionsPage.sitePlanning.description" demourl="https://carto.com/demo/siteplanning-demo/" infourl="https://carto.com/solutions/site-planning/"><template></template> <template></template> <template></template></solutioncard-stub>
      </li>
      <li class="grid-cell--col12 solution-element">
        <solutioncard-stub title="SolutionsPage.territoryManagement.title" description="SolutionsPage.territoryManagement.description" demourl="https://carto.com/demo/salesquest-demo/" infourl="https://carto.com/solutions/territory-management/"><template></template> <template></template> <template></template></solutioncard-stub>
      </li>
      <li class="grid-cell--col12 solution-element">
        <solutioncard-stub title="SolutionsPage.logisticsOptimization.title" description="SolutionsPage.logisticsOptimization.description" demourl="https://carto.com/demo/logistic-demo/" infourl="https://carto.com/solutions/logistics-optimization/"><template></template> <template></template> <template></template></solutioncard-stub>
      </li>
    </ul>
    <div class="talk-us-card">
      <h3 class="title is-medium">SolutionsPage.talkUsTitle</h3>
      <p class="text is-caption card-description">SolutionsPage.talkUsDescription</p>
      <div class="talk-us-button"><a href="https://carto.com/request-live-demo/" target="_blank" class="button u-no-decoration"><span>SolutionsPage.talkUsButton</span></a></div>
    </div>
  </div>
</section>
`;
